calico-enterprise/operations/ebpf/enabling-ebpf.mdx

```diff
@@ -221,5 +239,7 @@ To revert to standard Linux networking:
    ```bash
    kubectl patch ds -n kube-system kube-proxy --type merge -p '{"spec":{"template":{"spec":{"nodeSelector":{"non-calico": null}}}}}'
    ```
+2. If you are running MKE, follow the step procedure in [Modify an existing MKE configuration](https://docs.mirantis.com/mke/current/ops/administer-cluster/configure-an-mke-cluster/use-an-mke-configuration-file.html#modify-an-existing-mke-configuration) to download, edit, and upload your MKE configuration. During the editing step, add the following configuration:
+`kube_proxy_mode` to `iptables` and `kube_default_drop_masq_bits ` to `false`.
```

Comment:
> `kube_default_drop_masq_bits ` to `false`.

Are we sure that's the right default even for iptables mode?  Calico's default is to use 0xffff0000 bits in both modes but in BPF mode it cannot be changed.  So, if we have a conflict in BPF mode, we might have a conflict in iptables mode too?

Reply:
I checked with Vikram from Mirantis and he said its ok to set ```kube_default_drop_masq_bits``` to ```true``` even in iptables mode.

Reply:
Will make the changes in the doc accordingly.